src/holodeck/weather.py

```diff
@@ -84,23 +85,25 @@ def set_weather(self, weather_type):
         to the given weather.
 
         If there is no skysphere, skylight, or directional source light in the world, this command
-        will fail silently.
+        will crash the environment.
 
         ..note::
             Because this command can affect the fog density, any changes made by a
             ``change_fog_density`` command before a set_weather command called will be undone. It is
             recommended to call ``change_fog_density`` after calling set weather if you wish to
             apply your specific changes.
 
-        In all downloadable worlds, the weather is clear by default.
+        In all downloadable worlds, the weather is sunny by default.
 
         If the given type string is not available, the command will not be sent.
 
         Args:
-            weather_type (:obj:`str`): The type of weather, which can be ``rain`` or ``cloudy``.
+            weather_type (:obj:`str`): The type of weather, which can be ``rain``, ``cloudy``, or
+            ``sunny``.
 
         """
-        if not weather_type.lower() in ["rain", "cloudy"]:
+        if not weather_type.lower() in ["rain", "cloudy", "sunny"]:
             raise HolodeckException("Invalid weather type " + weather_type)
 
+        self.cur_weather = weather_type
```

Comment:
should we be saving `weather_type.lower()` since that is what we validated? Does the engine care?

Reply:
yes it does, nice catch.